cdm/radial/src/test/java/ucar/nc2/iosp/nids/TestNids.java

```diff
@@ -418,6 +418,94 @@ public void testRadialImageMessagePcode180() throws IOException {
     }
   }
 
+  @Test
+  public void testRadialImageMessagePcode153() throws IOException {
+    // Radial Image message, product code 153
+    double comparisonTolerance = 0.1;
+    String datafile = TestDir.localTestDataDir + "nids/H0Z_20200812_1318";
+    try (NetcdfFile ncf = NetcdfFiles.open(datafile)) {
+      Variable bref = ncf.findVariable("BaseReflectivityDR");
+      Array data = bref.read();
+      double max = MAMath.getMaximum(data);
+      // max reflectivity value as shown by NWS web display at the time
```

Comment:
Here and in the other tests:
1. These (with the exception of one test) are not reflectivity
2. Not how we arrived at these values 😉 

copy-pasta!

Reply:
Should be updated now. copy-pasta magic FTW!